d2renderers/d2sketch/sketch.go

```diff
@@ -53,78 +52,104 @@ func InitSketchVM() (*Runner, error) {
+	</pattern>`, luminanceCategory, fill)
 }
 
 func Rect(r *Runner, shape d2target.Shape) (string, error) {
 	js := fmt.Sprintf(`node = rc.rectangle(0, 0, %d, %d, {
-		fill: "%s",
-		stroke: "%s",
+		fill: "#000",
+		stroke: "#000",
 		strokeWidth: %d,
 		%s
-	});`, shape.Width, shape.Height, shape.Fill, shape.Stroke, shape.StrokeWidth, baseRoughProps)
+	});`, shape.Width, shape.Height, shape.StrokeWidth, baseRoughProps)
 	paths, err := computeRoughPathData(r, js)
 	if err != nil {
 		return "", err
 	}
 	output := ""
+	pathEl := svgstyle.NewThemableElement("path")
+	pathEl.Transform = fmt.Sprintf("translate(%d %d)", shape.Pos.X, shape.Pos.Y)
```

Comment:
this looks like it can just be a method now that you're applying it on a class.

`pathEl.Transform(shape.Pos.X, shape.Pos.Y)`, and then write the translate in Render()

Reply:
Makes sense, but a better name would be `SetTransformTranslate(shape.Pos.X, shape.Pos.Y)`

Reply:
SetTranslate should be enough. unambiguous

Reply:
sounds good to me